### PR TITLE
assoc, attrib, cd, choco-apikey, choco-feature, choco-info, choco, cls: add Japanese translation

### PR DESCRIPTION
**Pull request by nlsum1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Adding more Japanese translated commands under windows.

### CODE REVIEW

File: pages.ja/windows/attrib.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- ファイルに読み取り専用属性を追加します:
+
+`attrib +R {{資料.txt}}`
+
+- システムとファイルの非表示属性を削除します:
+
+`attrib -S -H {{資料.txt}}`
+
+- 非表示の属性をディレクトリに追加します:
```
Comment (navarroaxel) [inline pages.ja/windows/attrib.md:26]: I'm not sure about this translation for `Add the hidden attribute to a directory`:
Screenshot_20201010_093456

File: pages.ja/windows/attrib.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`attrib /S`
+
+- 現在のディレクトリとサブディレクトリ内のファイルとディレクトリの属性を表示します:
+
+`attrib /S /D`
+
+- ファイルに読み取り専用属性を追加します:
+
+`attrib +R {{資料.txt}}`
```
Comment (navarroaxel) [inline pages.ja/windows/attrib.md:20]: Are you sure about the translation for `document.txt`, you can use `filename.txt` too:
Screenshot_20201010_093841

File: pages.ja/windows/choco-apikey.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# choco-apikey
+
+> ChocolateyソースのAPIキーを管理します
+> 詳しくはこちら:  <https://chocolatey.org/docs/commands-apikey>.
```
Comment (navarroaxel) [inline pages.ja/windows/choco-apikey.md:4]: There are 2 spaces here:
```suggestion
> 詳しくはこちら: <https://chocolatey.org/docs/commands-apikey>.
```

File: pages.ja/windows/cd.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> 現在の作業ディレクトリの名前を表示するか、現在の作業ディレクトリを変更します
+> 詳しくはこちら:  <https://docs.microsoft.com/windows-server/administration/windows-commands/cd>.
```
Comment (navarroaxel) [inline pages.ja/windows/cd.md:4]:
```suggestion
> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/cd>.
```

File: pages.ja/windows/attrib.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# attrib
+
+> ファイルまたはディレクトリの属性を表示または変更します
+> 詳しくはこちら:  <https://docs.microsoft.com/windows-server/administration/windows-commands/attrib>.
```
Comment (navarroaxel) [inline pages.ja/windows/attrib.md:4]: There are 2 spaces here:
```suggestion
> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/attrib>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @nlsum1, thanks for your submission. I left some comments.

File: pages.ja/windows/attrib.md (hunk 1 above)
Comment (nlsum1) [reply pages.ja/windows/attrib.md:26]: By right "hidden attribute" (非表示の属性) in Japanese is not in plural form, and not supposed to be plural form even if retranslating back to English, if that is your concern.

File: pages.ja/windows/attrib.md (hunk 2 above)
Comment (nlsum1) [reply pages.ja/windows/attrib.md:20]: Yes, "filename" would make more sense, I have updated it accordingly.

**Review by navarroaxel — APPROVED** [review]
LGTM

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks, @name :slightly_smiling_face:

**Pull request merged**